### PR TITLE
Small bug fixes

### PR DESCRIPTION
**Pull request by jacquarg**
* Fix in WIFi handling : stop backup if wifi connection is lost (and wifi only option is set).
* Fix background service settings transmission : fix "image sync and service may not start".
* Add Error message with install cozy-files on 404 error on files routes.

Label : In progress (wait for new small bugs to fix)

### CODE REVIEW

**Comment by frankrousseau** [comment]
Be careful with tag label, it may be too big to be displayed in the phone UI.

**Comment by poupotte** [comment]
This PR is ready, @aenario , can you review it ?

**Comment by poupotte** [comment]
@frankrousseau , I'm not sure to understand, what do you mean by tag ?

**Comment by frankrousseau** [comment]
Don't worry, forget my comment.

File: www-src/app/replicator/replicator_backups.coffee [hunk 1]
```diff
@@ -170,7 +170,11 @@ module.exports =
                         else
                             # Create file
                             toUpload.push path
-                        setTimeout cb, 1
+
+                        DeviceStatus.checkReadyForSync (err, ready, msg) ->
+                            err = new Error msg unless ready or err?
```
Comment (aenario) [inline www-src/app/replicator/replicator_backups.coffee]: This line is hard to understand

File: www-src/app/replicator/replicator_backups.coffee [hunk 2]
```diff
@@ -184,7 +188,11 @@ module.exports =
                     console.log "UPLOADING #{path}"
                     @uploadPicture path, device, (err) =>
                         console.log "ERROR #{path} #{err}" if err
-                        setTimeout cb, 1
+                        DeviceStatus.checkReadyForSync (err, ready, msg) ->
+                            err = new Error msg unless ready or err?
```
Comment (aenario) [inline www-src/app/replicator/replicator_backups.coffee]: idem

File: www-src/app/replicator/replicator_backups.coffee [hunk 3]
```diff
@@ -170,7 +170,11 @@ module.exports =
                         else
                             # Create file
                             toUpload.push path
-                        setTimeout cb, 1
+
+                        DeviceStatus.checkReadyForSync (err, ready, msg) ->
+                            err = new Error msg unless ready or err?
+                            return cb err if err
+                            setTimeout cb, 1
```
Comment (aenario) [inline www-src/app/replicator/replicator_backups.coffee]: comment on why setTimeout

**Pull request merged**